### ISSUE TITLE
Mac/Linux deploy failing

### ISSUE THREAD

**Issue by squeek502** [luvit/luvi#225]
https://travis-ci.org/luvit/luvi/builds/641610846

Not really sure what's going wrong. Is it this?

```
Fetching submodule deps/lua-openssl
Fetching submodule deps/luv
fatal: remote error: upload-pack: not our ref b0e9de051656471a8e12bf1e576285642b02c3e1c886cb4ade983dadedc721d9f0071924d5aee
From https://github.com/zhaozg/lua-openssl
 * branch            80896f0f3914dffe4228d8d8dd7e2c68e329c88c -> FETCH_HEAD
The command "git fetch --unshallow" exited with 1.
```

On Linux there's also this:

```
In file included from /luvi/deps/luv/deps/luajit/src/ljamalg.c:68:0:
/luvi/deps/luv/deps/luajit/src/lj_opt_fold.c: In function 'lj_opt_fold':
/luvi/deps/luv/deps/luajit/src/lj_opt_fold.c:2475:18: warning: implicit declaration of function 'fold_hashkey'; did you mean 'hashkey'? [-Wimplicit-function-declaration]
     uint32_t h = fold_hashkey(k);
                  ^~~~~~~~~~~~
                  hashkey
/luvi/deps/luv/deps/luajit/src/lj_opt_fold.c:2476:19: error: 'fold_hash' undeclared (first use in this function); did you mean 'fold_prof'?
     uint32_t fh = fold_hash[h];  /* Lookup key in semi-perfect hash table. */
                   ^~~~~~~~~
                   fold_prof
/luvi/deps/luv/deps/luajit/src/lj_opt_fold.c:2476:19: note: each undeclared identifier is reported only once for each function it appears in
In file included from /luvi/deps/luv/deps/luajit/src/lj_jit.h:10:0,
                 from /luvi/deps/luv/deps/luajit/src/lj_trace.h:12,
                 from /luvi/deps/luv/deps/luajit/src/lj_gc.c:27,
                 from /luvi/deps/luv/deps/luajit/src/ljamalg.c:31:
/luvi/deps/luv/deps/luajit/src/lj_opt_fold.c:2478:29: error: 'fold_func' undeclared (first use in this function); did you mean 'FoldFunc'?
       ref = (IRRef)tref_ref(fold_func[fh >> 24](J));
                             ^
/luvi/deps/luv/deps/luajit/src/lj_ir.h:487:33: note: in definition of macro 'tref_ref'
 #define tref_ref(tr)  ((IRRef1)(tr))
                                 ^~
```

Only happens during deploy, not during the regular CI steps.

**Comment by zhaozg** [comment]
I will look into it

**Comment by squeek502** [comment]
This should fix the Linux error, not sure about Mac: https://github.com/luvit/luv/pull/454

**Comment by squeek502** [comment]
Fixed by luvit/luv#454 / #226
